src/pages/Navgurukul/merakiAdmission/index.js

```diff
@@ -0,0 +1,211 @@
+        }
+      );
+      return;
+    }
+    if (userDetails.mobileNumber.toString().length !== 10) {
+      toast.error("Please give 10 digits of the mobile number.", {
+        position: toast.POSITION.BOTTOM_RIGHT,
+      });
+      return;
+    }
+    await isDuplicate();
+  };
+
+  const isDuplicate = () => {
+    const { firstName, middleName, lastName, mobileNumber } = userDetails;
+    axios
+      .get(`${process.env.REACT_APP_CHANAKYA_BASE_URL}/check_duplicate`, {
+        params: {
+          Name: firstName.concat(" ", middleName, lastName),
+          Number: mobileNumber,
+        },
+      })
+      .then(async (data) => {
+        const response = data.data.data;
+        const stage = response.pendingInterviewStage;
+        const url = `https://admissions.navgurukul.org/check_duplicate/Name=${firstName}${middleName}${lastName}&Number=${mobileNumber}&Stage=${stage}`;
+        if (response.alreadyGivenTest) {
+          window.open(url, "_blank");
+        } else {
+          generateTestLink();
+        }
+      });
+  };
+
+  const changeHandler = (e) => {
+    setUserDetails({ ...userDetails, [e.target.name]: e.target.value });
+  };
+
+  return (
+    <div className="admission">
+      <div className="left-section">
+        <YouTube className={"video-alumni"} videoId={`vuSwndj5cbs`} />;
+        <div className="video-label">Experience of NG Alumni & Graduates</div>
+      </div>
+      <div className="right-section">
+        <div className="test-form">
+          <h2 className="scholarship-test">
+            Software Engineering Scholarship Test
+          </h2>
+          <div className="input-form">
+            <div className="form-row">
+              <div className="input-field-test">
+                <span>First Name</span>
+                <input
+                  type="text"
+                  placeholder="Abhi..."
+                  value={userDetails.firstName}
+                  name="firstName"
+                  onChange={changeHandler}
+                />
+              </div>
+              <div className="input-field-test">
+                <span>Middle Name (Optional)</span>
+                <input
+                  type="text"
+                  placeholder="Kumar..."
+                  value={userDetails.middleName}
+                  name="middleName"
+                  onChange={changeHandler}
+                />
+              </div>
+              <div className="input-field-test">
+                <span>Last Name</span>
+                <input
+                  type="text"
+                  placeholder="Garg..."
+                  value={userDetails.lastName}
+                  name="lastName"
+                  onChange={changeHandler}
+                />
+              </div>
+            </div>
+            <div className="form-row">
+              <div className="input-field-test">
+                <span>Mobile Number</span>
+                <input
+                  type="text"
```

Comment:
You can use the `tel` input type with pattern validation here to require 10-digit phone numbers (See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/tel#pattern_validation) as follows.  However, the pattern for the input will not be enforced here as the inputs are not inside a `<form>` element.  

It may be useful to create a `PhoneNumberInput` component to remove hyphens and parentheses, look up entered phone numbers to make sure they're valid, etc.

```suggestion
                  type="tel"
                  pattern="^[0-9]{10}$"
```

Reply:
@jschanker even after using patter with a limit of 10, we can use more than 10, so why do we need to use pattern? 
Also `type='tel'` can take alphabets, so why need to use it, we can use `type='number'` instead?

Reply:
@Poonam-Singh-Bagh Using `type='tel'` gives the browser (and search engines) better semantic information about the form element.  If you use `type = 'number'`, then you may have up/down arrows on the right of the form field, which could be confusing to the user as you don't usually enter telephone numbers that way.  As for `pattern`, this again provides more information to the browser.  If you have the following HTML, then modern browsers will often prevent the form from being submitted when the user enters a number that isn't exactly 10 digits with a validation message appearing under the text box, saying something like, "Please match the requested format."  This message will pop up only *after* you click the submit button and will not occur if the input appears outside a form.  However, even in cases where it has no functional purpose for a browser, search engines can still make use of this extra information.  Also, I updated the regex with the `^` and `$`, as you had mentioned, to make sure it wasn't matching a proper substring of an invalid input.  Sorry about my mistake before.
```
<form>
  <input type="tel" pattern="^[0-9]{10}$" />
  <button type="submit">Submit</button>
</form>
```